oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollectorIT.java

```diff
@@ -2649,7 +2647,6 @@ public void testDeletedPropsAndUnmergedBCWithCollisionWithDryRunMode() throws Ex
         RevisionVector br3 = unmergedBranchCommit(store1, b -> b.child("foo").setProperty("a", "d"));
         RevisionVector br4 = unmergedBranchCommit(store1, b -> b.child("bar").setProperty("x", "z"));
         mergedBranchCommit(store1, b -> b.child("foo").removeProperty("p"));
-        store1.runBackgroundOperations();
```

Comment:
AFAIK, without running the background operations, the changes won't be visible to other clusters.
Why did you commented it out ?

Reply:
Yes, indeed, commenting out that line will one hide the underlying issue. We have a subtle race condition in fullGC  FullGCMode.ORPHANS_EMPTYPROPS_KEEP_ONE_ALL_PROPS(it appears randomly), we will fix the issue and let this test as it is.